designs/lily/src/front.mjs

```diff
@@ -0,0 +1,610 @@
+
+    paths.bottom = new Path().move(points.bottomIn).line(points.bottomOut)
+
+    // note: upperInseam contains waist and cross seam as well
+    paths.upperInseam = drawInseam()
+      .curve(points.crotchSeamCurveCp1, points.crotchSeamCurveCp2, points.crotchSeamCurveStart)
+      .line(points.styleWaistIn)
+      .line(points.styleWaistOut)
+    paths.upperOutseam = drawOutseam()
+    paths.bottom.hide()
+    paths.upperInseam.hide()
+    paths.upperOutseam.hide()
+  }
+
+  if (complete) {
+    points.grainlineTop.y = points.styleWaistInLily.y
+    points.grainlineBottom.y = points.bottom.y
+    macro('grainline', {
+      from: points.grainlineTop,
+      to: points.grainlineBottom,
+    })
+    points.logoAnchor = new Point(points.crotchSeamCurveStart.x / 2, points.fork.y)
+    snippets.logo = new Snippet('logo', points.logoAnchor)
+    points.titleAnchor = points.logoAnchor.shift(-90, 60)
+    macro('title', {
+      nr: 2,
+      title: 'front',
+      at: points.titleAnchor,
+    })
+
+    //notches
+    if (options.fitGuides) {
+      points.waistMid = points.waistOut.shiftFractionTowards(points.waistIn, 0.5)
+      points.seatMid = points.waistMid
+        .shiftTowards(points.waistOut, measurements.waistToSeat)
+        .rotate(90, points.waistMid)
+      points.seatInTarget = points.seatOut.shiftOutwards(points.seatMid, measurements.seat / 4)
+      points.seatOutTarget = points.seatMid.shiftTowards(points.seatOut, measurements.seat / 4)
+      points.hipsInTarget = points.waistIn
+        .shiftTowards(points.waistOut, measurements.waistToHips)
+        .rotate(90, points.waistIn)
+      points.hipsOutTarget = points.waistOut
+        .shiftTowards(points.waistIn, measurements.waistToHips)
+        .rotate(-90, points.waistOut)
+      points.hipsIn = utils.beamsIntersect(
+        points.hipsOutTarget,
+        points.hipsInTarget,
+        points.waistIn,
+        points.crotchSeamCurveStart
+      )
+      // intersection between the vertical line from waistMid to seatMid and a line parallel to waistIn-waistOut that goes through crotchSeamCurveStart
+      points.crotchSeamCurveStartMid = utils.beamsIntersect(
+        points.crotchSeamCurveStart,
+        points.crotchSeamCurveStart.shift(points.waistIn.angle(points.waistOut), 1),
+        points.waistMid,
+        points.seatMid
+      )
+      // check whether intersection occurs above or below crotch seam curve start
+      points.seatInTemp = utils.beamsIntersect(
+        points.seatMid,
+        points.seatInTarget,
+        points.crotchSeamCurveStart,
+        points.waistIn
+      ) // NOTE: guaranteed to return a Point since the lines cannot be parallel
+      if (points.seatInTemp.y <= points.crotchSeamCurveStartMid.y) {
+        // intersection is above the crotch seam curve start, so on the line segment
+        points.seatIn = points.seatInTemp.clone()
+      } else if (points.seatInTemp.y > points.fork) {
+        // seat appears to be below crotch
```

Comment:
There's a lot of commented out code here, along with a `TODO: fix this`. Should we be concerned?

Reply:
It appears to work well enough so far, so I've restored the warning but removed the commented-out code.